sw/device/tests/pmp_smoketest_tor.c

```diff
@@ -47,13 +47,11 @@ static volatile char pmp_load_test_data[PMP_LOAD_RANGE_BUFFER_SIZE];
 
 static uint32_t get_mepc(void) {
   uint32_t mepc;
-  asm volatile("csrr %0, mepc" : "=r"(mepc) :);
+  CSR_READ(CSR_REG_MEPC, &mepc);
   return mepc;
 }
 
-static void set_mepc(uint32_t mepc) {
-  asm volatile("csrw mepc, %0" : : "r"(mepc));
-}
+static void set_mepc(uint32_t mepc) { CSR_WRITE(CSR_REG_MEPC, mepc); }
```

Comment:
is clang-format happy with this?

Reply:
Yeah, I think clang-format actually made that change (I'm not really a fan of single line function definitions unless there is a specific reason to do so).

Reply:
Yeah. clang-format in Google style is *super* excited about one-line functions. Do not like.

I bet we can fix it if we fiddle with our clang format config if we care enough...